### DISCUSSION TITLE
Example expo router app to share (stacked navigations in tab bar)

### DISCUSSION THREAD

**Discussion by SKempin** [expo/expo#32941] [Proposals and Ideas]
Having struggled understanding the expo router documentation I have created an example app that demonstrates nested stacks in  a tab bar. 

I think this would actually be very useful to share with the community (I found multiple stack overflow and github issues discussing problems with creating this layout).

Where is the best place to share this repo and link to it? Ideally it would go in the official docs but sharing an external repo doesn't seem to be supported. I would happily upload the repo to another repo if required.

**Comment by sgandon** [comment]
do you have a link to you example, indeed as a newbie I'd be interested in such examples.
